cpp/lib/ResultSetArrow.cpp

```diff
@@ -44,6 +43,36 @@ ResultSetArrow::ResultSetArrow(
     m_currRowIdx = 0;
 }
 
+ResultSetArrow::ResultSetArrow(
+    cJSON * jsonRowset64,
+    SF_COLUMN_DESC * metadata,
+    std::string tzString
+) :
+    ResultSet(metadata, tzString)
+{
+    m_queryResultFormat = SF_ARROW_FORMAT;
```

Comment:
can we pass the format in base class constructor above?

Reply:
done